ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,126 @@
     "FieldType": "Document",
     "ElementLabel": "Upload a file",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "EmailAddress",
+    "ListElementCode": "email",
+    "FieldType": "Text",
+    "ElementLabel": "Email",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Telephone",
+    "ListElementCode": "telephoneNumber",
+    "FieldType": "Text",
+    "ElementLabel": "Telephone",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Telephone",
+    "ListElementCode": "contactDirection",
+    "FieldType": "Text",
+    "ElementLabel": "Person to contact",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "MobileNumber",
```

Comment:
In my mind MobileNumber shouldn't be a separate Type to telephone number. What are your thoughts? There is a field in [Telephone](https://tools.hmcts.net/confluence/display/RTA/Telephone+Number) called telephoneUsageType which should differentiate this

Reply:
Again I am going by the schema structure I was given. I feel like you should have a distinctive mobile number rather than seeing telephone that is of mobile type but im not up to date with business decisions so I will go with what you think?

Reply:
ref: https://tools.hmcts.net/jira/browse/FPLA-623